CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+
+- Added `IpAddress` GraphQL custom scalar for IP addresses.
+  - Applied it to the GraphQL APIs `ipLocation`, `ipLocationList`,
+    `insertAccount`, `updateAccount`, `insertSamplingPolicy`, and
+    `updateSamplingPolicy`.
+  - The API returns a specific error message if the value cannot be parsed
+    into `IpAddr`.
+    For example: `Failed to parse "IpAddress": Invalid IP address: abc(occurred
+while parsing "[IpAddress!]")`
```

Comment:
For clarity, I suggest:

The API returns the following error message when a value cannot be parsed as an `IpAddr`: `Failed to parse "IpAddress": Invalid IP address: abc (occurred while parsing "[IpAddress!]")`, where "abc" is the input.